mlflow/server/handlers.py

```diff
@@ -799,6 +818,52 @@ def _delete_model_version_tag():
     return _wrap_response(DeleteModelVersionTag.Response())
 
 
+@catch_mlflow_exception
+def _download_artifact(artifact_path):
+    basename = os.path.basename(artifact_path)
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        tmp_path = os.path.join(tmp_dir, basename)
+        artifact_repo = _get_artifact_repo_mlflow_artifacts()
+        artifact_repo._download_file(artifact_path, tmp_path)
+        return send_from_directory(tmp_dir, basename)
+
+
+@catch_mlflow_exception
+def _upload_artifact(artifact_path):
+    head, tail = os.path.split(artifact_path)
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        tmp_path = os.path.join(tmp_dir, tail)
+        with open(tmp_path, "wb") as f:
+            chunk_size = 8192
```

Comment:
Where are we getting the chunk size of 3KB from? Should we potentially choose a larger chunk size?

Reply:
It's just a randomly-selected value. Using a larger chunk size does make sense.

Reply:
Let's use a larger size! I think most network connections can handle it :D